### ISSUE TITLE
how to use keras tuner for tuning batch size and number of epochs?

### ISSUE THREAD

**Issue by inginfor** [keras-team/keras-tuner#1042]
In case you need to optimize batch size, number of epochs, learning rate, type of optimizer and dropout rate how should you do it?
